src/Header/APIHandler.h

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) 2021, 鍾淯丞, 周杰仕, 林仁鴻. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include <string>
+#include "./OperateData.h"
+
+class APIHandler
+{
+
+public:
```

Comment:
1. Missing constructor.
2. Is this copyable? If not, please delete copy constructor.

Reply:
noncopyable  constructor added